### PR TITLE
Initial commit

### LINKED ISSUE #1: Initialize project
As a developer I'd like to have this project initialized.

### PR DESCRIPTION
**Pull request by villejuutila**
Resolves #1

### CODE REVIEW

**Review by EVP1989 — APPROVED** [review]
approved

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
-# noheva-visitor-ui
+# noheva_visitor_ui
+
+A new Flutter project.
```
Comment (villejuutila) [inline README.md]: Add a short description of the application.

File: analysis_options.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  # Instead of disabling a lint rule for the entire project in the
+  # section below, it can also be suppressed for a single line of code
+  # or a specific dart file by using the `// ignore: name_of_lint` and
+  # `// ignore_for_file: name_of_lint` syntax on the line or in the file
+  # producing the lint.
+  rules:
+    avoid_print: true
+    prefer_single_quotes: false
+# Additional information about this file can be found at
+# https://dart.dev/guides/language/analysis-options
```
Comment (villejuutila) [inline analysis_options.yaml]:
```suggestion
include: package:flutter_lints/flutter.yaml

linter:
  # A list of all available lints
  # and their documentation is published at
  # https://dart-lang.github.io/linter/lints/index.html.
  rules:
    avoid_print: true
    prefer_single_quotes: false
# Additional information about this file can be found at
# https://dart.dev/guides/language/analysis-options
```

File: pubspec.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+name: noheva_visitor_ui
+description: noheva_visitor_ui.
```
Comment (villejuutila) [inline pubspec.yaml]: Add a short description of the application. And by short I mean short :)

**Pull request merged**